akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -270,6 +297,26 @@ class ClusterClient(settings: ClusterClientSettings) extends Actor with Stash wi
     else if (contacts.size == 1) (initialContactsSel ++ contacts) foreach { _ ! GetContacts }
     else contacts foreach { _ ! GetContacts }
   }
+
+  def buffer(msg: Any): Unit =
```

Comment:
I almost don't read Scala, but will it be possible (in Java) to override the method 'buffer' and 'sendBuffer'?

What i am trying to achieve is the following: I have GPS updates from many devices, and instead of throwing away the oldest, I could throw away older messages from the same device. That is, replacing 'buffer' with a map mapping from device ID to a list of messages (or the last one, if I only care about that one).